toolset/setup/linux/frameworks/cppcms-cppdb.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+fw_installed cppcms-cppdb && return 0
+fw_depends cppcms
+
+#HOW THAT GOES
+#http://cppcms.com/sql/cppdb/build.html
+
+VERSION=0.3.1
+BACKNAME=cppdb
+CPPCMS_HOME=$IROOT/$BACKNAME-$VERSION
+
+sudo apt-get update -qq
+#unixodbc-dev libsqlite3-dev
+sudo apt-get install -y cmake libmysqld-dev libpq-dev
```

Comment:
I believe you can drop this line and instead hoist those libs up to the `fw_depends` line; it should check locally, and if no file exists (`cmake.sh` for example) it will fallback to `apt-get`.